.travis.yml

```diff
@@ -8,8 +6,33 @@ services:
 go:
   - "1.13.6"  # Keep in sync with Makefile and README.md
 
-os:
-  - linux
+arch:
+  - amd64
+  - arm64
+
+
+dist: bionic
+os: linux
+
+jobs:
+  include:
+   - os: linux
+     arch: amd64
+   - os: linux
+     arch: arm64
+   - stage: Push Docker manifest
+     env:
+       - DOCKER_CLI_EXPERIMENTAL=enabled
+     script:
+       - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+       - sudo add-apt-repository "deb [arch=$TRAVIS_CPU_ARCH] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
```

Comment:
What is the reason we need to add in our docker client?

Reply:
Default Docker client version is very old, 18.06, there has been many bug fixes around buildkit since. I also changed Ubuntu version by `dist: bionic`, to get the newer Docker version.

I didn't hit particular issue for this change, but I did attempt to use `buildx`, which is not supported in v18.06. In general, I think it's better to use a newer version of Docker.